daemon/logger/journald/read.go

```diff
@@ -125,10 +125,10 @@ func (s *journald) Close() error {
 	return nil
 }
 
-// convert error code returned from a sd_journal_* function
+// CErr converts error code returned from a sd_journal_* function
 // (which returns -errno) to a string
 func CErr(ret C.int) string {
-	return C.GoString(C.strerror(C.int(-ret)))
+	return C.GoString(C.strerror(C.int(-ret))) //nolint: unconvert
```

Comment:
Why wouldn't we write `C.GoString(C.strerror(-ret))`?  `ret` is already a `C.int`.

Reply:
Hm... good call. I hate CGO, and probably wasn't sure if it was safe to remove the casting; let me update 👍